R/install-git.r

```diff
@@ -69,3 +76,44 @@ remote_metadata.git_remote <- function(x, bundle = NULL, source = NULL) {
     RemoteSha = sha
   )
 }
+
+remote_sha.git_remote <- function(remote, ...) {
+  res <- system(
```

Comment:
https://github.com/ropensci/git2r/issues/130 would seem to suggest no, I can try and hunt down the code in libgit2 and see how much work it would be to add to git2r